paasta_tools/cli/cmds/status.py

```diff
@@ -831,6 +861,20 @@ def print_kubernetes_status(
             )
         )
     )
+    if kubernetes_status.create_timestamp:
```

Comment:
It seems you are assuming create_timestamp exists, so this if condition is unnecessary?  I'm not sure if create_timestamp is always available on k8s api. So I think it's safer to make it optional here:     kstatus["create_timestamp"] = app.metadata.creation_timestamp.timestamp()

Reply:
The thing I worried about here is that if we make new paasta release, there's no guarantee that server side would be upgraded first. If the client side upgrades first without this condition, it would throw errors because of NoneType. At least from the my testing, create_timestamp is always available on k8s api. Should we make this optional? It would mask the errors imo.

Reply:
@chlgit good point. I guess you can make another PR to remove this line after it's release. Leave it as it is for now.